### ISSUE TITLE
2025-04-08, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#134]
# Today's Arxiv Papers

(1) [URECA: Unique Region Caption Anything](http://arxiv.org/abs/2504.05305v1), 2025-04-07

(2) [Truthful or Fabricated? Using Causal Attribution to Mitigate Reward Hacking in Explanations](http://arxiv.org/abs/2504.05294v1), 2025-04-07

(3) [The challenge of uncertainty quantification of large language models in medicine](http://arxiv.org/abs/2504.05278v1), 2025-04-07

(4) [Enhancing LLM-Based Short Answer Grading with Retrieval-Augmented Generation](http://arxiv.org/abs/2504.05276v1), 2025-04-07

(5) [Do PhD-level LLMs Truly Grasp Elementary Addition? Probing Rule Learning vs. Memorization in Large Language Models](http://arxiv.org/abs/2504.05262v1), 2025-04-07

(6) [How to evaluate control measures for LLM agents? A trajectory from today to superintelligence](http://arxiv.org/abs/2504.05259v1), 2025-04-07

(7) [Learning to Reason Over Time: Timeline Self-Reflection for Improved Temporal Reasoning in Language Models](http://arxiv.org/abs/2504.05258v1), 2025-04-07

(8) [Explaining Low Perception Model Competency with High-Competency Counterfactuals](http://arxiv.org/abs/2504.05254v1), 2025-04-07

(9) [LLM-based Automated Grading with Human-in-the-Loop](http://arxiv.org/abs/2504.05239v1), 2025-04-07

(10) [Mapping biodiversity at very-high resolution in Europe](http://arxiv.org/abs/2504.05231v1), 2025-04-07

(11) [Leveraging LLMs for Utility-Focused Annotation: Reducing Manual Effort for Retrieval and RAG](http://arxiv.org/abs/2504.05220v1), 2025-04-07

(12) [LLM-Alignment Live-Streaming Recommendation](http://arxiv.org/abs/2504.05217v1), 2025-04-07

(13) [Unleashing the Power of LLMs in Dense Retrieval with Query Likelihood Modeling](http://arxiv.org/abs/2504.05216v1), 2025-04-07

(14) [Post-Training Language Models for Continual Relation Extraction](http://arxiv.org/abs/2504.05214v1), 2025-04-07

(15) [Quantum Program Linting with LLMs: Emerging Results from a Comparative Study](http://arxiv.org/abs/2504.05204v1), 2025-04-07

(16) [Concise Reasoning via Reinforcement Learning](http://arxiv.org/abs/2504.05185v1), 2025-04-07

(17) [BRIDGES: Bridging Graph Modality and Large Language Models within EDA Tasks](http://arxiv.org/abs/2504.05180v1), 2025-04-07

(18) [Evaluating Knowledge Graph Based Retrieval Augmented Generation Methods under Knowledge Incompleteness](http://arxiv.org/abs/2504.05163v1), 2025-04-07

(19) [PanoDreamer: Consistent Text to 360-Degree Scene Generation](http://arxiv.org/abs/2504.05152v1), 2025-04-07

(20) [Pr$εε$mpt: Sanitizing Sensitive Prompts for LLMs](http://arxiv.org/abs/2504.05147v1), 2025-04-07

(21) [Query Smarter, Trust Better? Exploring Search Behaviours for Verifying News Accuracy](http://arxiv.org/abs/2504.05146v1), 2025-04-07

(22) [DoCIA: An Online Document-Level Context Incorporation Agent for Speech Translation](http://arxiv.org/abs/2504.05122v1), 2025-04-07

(23) [Algorithm Discovery With LLMs: Evolutionary Search Meets Reinforcement Learning](http://arxiv.org/abs/2504.05108v1), 2025-04-07

(24) [AI for Climate Finance: Agentic Retrieval and Multi-Step Reasoning for Early Warning System Investments](http://arxiv.org/abs/2504.05104v1), 2025-04-07

(25) [Speech-to-Trajectory: Learning Human-Like Verbal Guidance for Robot Motion](http://arxiv.org/abs/2504.05084v1), 2025-04-07

(26) [The Curse of CoT: On the Limitations of Chain-of-Thought in In-Context Learning](http://arxiv.org/abs/2504.05081v1), 2025-04-07

(27) [On the Performance of an Explainable Language Model on PubMedQA](http://arxiv.org/abs/2504.05074v1), 2025-04-07

(28) [Not All Data Are Unlearned Equally](http://arxiv.org/abs/2504.05058v1), 2025-04-07

(29) [Revealing the Intrinsic Ethical Vulnerability of Aligned Large Language Models](http://arxiv.org/abs/2504.05050v1), 2025-04-07

(30) [Debate Only When Necessary: Adaptive Multiagent Collaboration for Efficient LLM Reasoning](http://arxiv.org/abs/2504.05047v1), 2025-04-07

(31) [InstructionBench: An Instructional Video Understanding Benchmark](http://arxiv.org/abs/2504.05040v1), 2025-04-07

(32) [Mixture-of-Personas Language Models for Population Simulation](http://arxiv.org/abs/2504.05019v1), 2025-04-07

(33) [Surveying Professional Writers on AI: Limitations, Expectations, and Fears](http://arxiv.org/abs/2504.05008v1), 2025-04-07

(34) [Enhancing Smart Contract Vulnerability Detection in DApps Leveraging Fine-Tuned LLM](http://arxiv.org/abs/2504.05006v1), 2025-04-07

(35) [Following the Whispers of Values: Unraveling Neural Mechanisms Behind Value-Oriented Behaviors in LLMs](http://arxiv.org/abs/2504.04994v1), 2025-04-07

(36) [A Domain-Based Taxonomy of Jailbreak Vulnerabilities in Large Language Models](http://arxiv.org/abs/2504.04976v1), 2025-04-07

(37) [Towards Visual Text Grounding of Multimodal Large Language Model](http://arxiv.org/abs/2504.04974v1), 2025-04-07

(38) [The Dream Within Huang Long Cave: AI-Driven Interactive Narrative for Family Storytelling and Emotional Reflection](http://arxiv.org/abs/2504.04968v1), 2025-04-07

(39) [M-Prometheus: A Suite of Open Multilingual LLM Judges](http://arxiv.org/abs/2504.04953v1), 2025-04-07

(40) [A Unified Pairwise Framework for RLHF: Bridging Generative Reward Modeling and Policy Optimization](http://arxiv.org/abs/2504.04950v1), 2025-04-07

(41) [A Llama walks into the 'Bar': Efficient Supervised Fine-Tuning for Legal Reasoning in the Multi-state Bar Exam](http://arxiv.org/abs/2504.04945v1), 2025-04-07

(42) [Lemmanaid: Neuro-Symbolic Lemma Conjecturing](http://arxiv.org/abs/2504.04942v1), 2025-04-07

(43) [Collab-RAG: Boosting Retrieval-Augmented Generation for Complex Question Answering via White-Box and Black-Box LLM Collaboration](http://arxiv.org/abs/2504.04915v1), 2025-04-07

(44) [Video-Bench: Human-Aligned Video Generation Benchmark](http://arxiv.org/abs/2504.04907v1), 2025-04-07

(45) [SCAM: A Real-World Typographic Robustness Evaluation for Multimodal Foundation Models](http://arxiv.org/abs/2504.04893v1), 2025-04-07

(46) [Leveraging Large Language Models for Cost-Effective, Multilingual Depression Detection and Severity Assessment](http://arxiv.org/abs/2504.04891v1), 2025-04-07

(47) [SoK: LLM-based Log Parsing](http://arxiv.org/abs/2504.04877v1), 2025-04-07

(48) [Simulating Persuasive Dialogues on Meat Reduction with Generative Agents](http://arxiv.org/abs/2504.04872v1), 2025-04-07

(49) [BIASINSPECTOR: Detecting Bias in Structured Data through LLM Agents](http://arxiv.org/abs/2504.04855v1), 2025-04-07

(50) [Quantization Hurts Reasoning? An Empirical Study on Quantized Reasoning Models](http://arxiv.org/abs/2504.04823v1), 2025-04-07

(51) [Beyond Answers: How LLMs Can Pursue Strategic Thinking in Education](http://arxiv.org/abs/2504.04815v1), 2025-04-07

(52) [Select Me! When You Need a Tool: A Black-box Text Attack on Tool Selection](http://arxiv.org/abs/2504.04809v1), 2025-04-07

(53) [ELT-Bench: An End-to-End Benchmark for Evaluating AI Agents on ELT Pipelines](http://arxiv.org/abs/2504.04808v1), 2025-04-07

(54) [OrderChain: A General Prompting Paradigm to Improve Ordinal Understanding Ability of MLLM](http://arxiv.org/abs/2504.04801v1), 2025-04-07

(55) [Weak-for-Strong: Training Weak Meta-Agent to Harness Strong Executors](http://arxiv.org/abs/2504.04785v1), 2025-04-07

(56) [OCC-MLLM-CoT-Alpha: Towards Multi-stage Occlusion Recognition Based on Large Language Models via 3D-Aware Supervision and Chain-of-Thoughts Guidance](http://arxiv.org/abs/2504.04781v1), 2025-04-07

(57) [Improving Multilingual Retrieval-Augmented Language Models through Dialectic Reasoning Argumentations](http://arxiv.org/abs/2504.04771v1), 2025-04-07

(58) [Can LLMs Interpret and Leverage Structured Linguistic Representations? A Case Study with AMRs](http://arxiv.org/abs/2504.04745v1), 2025-04-07

(59) [Enhancing Compositional Reasoning in Vision-Language Models with Synthetic Preference Data](http://arxiv.org/abs/2504.04740v1), 2025-04-07

(60) [TathyaNyaya and FactLegalLlama: Advancing Factual Judgment Prediction and Explanation in the Indian Legal Context](http://arxiv.org/abs/2504.04737v1), 2025-04-07

(61) [Synthetic Data Generation & Multi-Step RL for Reasoning & Tool Use](http://arxiv.org/abs/2504.04736v1), 2025-04-07

(62) [Can LLM-Driven Hard Negative Sampling Empower Collaborative Filtering? Findings and Potentials](http://arxiv.org/abs/2504.04726v1), 2025-04-07

(63) [Beyond Single-Turn: A Survey on Multi-Turn Interactions with Large Language Models](http://arxiv.org/abs/2504.04717v1), 2025-04-07

(64) [Are You Getting What You Pay For? Auditing Model Substitution in LLM APIs](http://arxiv.org/abs/2504.04715v1), 2025-04-07

(65) [Sequential-NIAH: A Needle-In-A-Haystack Benchmark for Extracting Sequential Needles from Long Contexts](http://arxiv.org/abs/2504.04713v1), 2025-04-07

(66) [Generalising from Self-Produced Data: Model Training Beyond Human Constraints](http://arxiv.org/abs/2504.04711v1), 2025-04-07

(67) [LagKV: Lag-Relative Information of the KV Cache Tells Which Tokens Are Important](http://arxiv.org/abs/2504.04704v1), 2025-04-07

(68) [Causal Retrieval with Semantic Consideration](http://arxiv.org/abs/2504.04700v1), 2025-04-07

(69) [R2Vul: Learning to Reason about Software Vulnerabilities with Reinforcement Learning and Structured Reasoning Distillation](http://arxiv.org/abs/2504.04699v1), 2025-04-07

(70) [scAgent: Universal Single-Cell Annotation via a LLM Agent](http://arxiv.org/abs/2504.04698v1), 2025-04-07

(71) [Generative Large Language Model usage in Smart Contract Vulnerability Detection](http://arxiv.org/abs/2504.04685v1), 2025-04-07

(72) [ACE-RLHF: Automated Code Evaluation and Socratic Feedback Generation Tool using Large Language Models and Reinforcement Learning with Human Feedback](http://arxiv.org/abs/2504.04657v1), 2025-04-07

(73) [LEO-MINI: An Efficient Multimodal Large Language Model using Conditional Token Reduction and Mixture of Multi-Modal Experts](http://arxiv.org/abs/2504.04653v1), 2025-04-07

(74) [Autono: ReAct-Based Highly Robust Autonomous Agent Framework](http://arxiv.org/abs/2504.04650v1), 2025-04-07

(75) [Splits! A Flexible Dataset for Evaluating a Model's Demographic Social Inference](http://arxiv.org/abs/2504.04640v1), 2025-04-06

(76) [Foundation Models for Software Engineering of Cyber-Physical Systems: the Road Ahead](http://arxiv.org/abs/2504.04630v1), 2025-04-06

(77) [Exact Unlearning of Finetuning Data via Model Merging at Scale](http://arxiv.org/abs/2504.04626v1), 2025-04-06

(78) [Capturing AI's Attention: Physics of Repetition, Hallucination, Bias and Beyond](http://arxiv.org/abs/2504.04600v1), 2025-04-06

(79) [SECQUE: A Benchmark for Evaluating Real-World Financial Analysis Capabilities](http://arxiv.org/abs/2504.04596v1), 2025-04-06

(80) [Hierarchical Planning for Complex Tasks with Knowledge Graph-RAG and Symbolic Verification](http://arxiv.org/abs/2504.04578v1), 2025-04-06

(81) [KnowsLM: A framework for evaluation of small language models for knowledge augmentation and humanised conversations](http://arxiv.org/abs/2504.04569v1), 2025-04-06

(82) [Planning Safety Trajectories with Dual-Phase, Physics-Informed, and Transportation Knowledge-Driven Large Language Models](http://arxiv.org/abs/2504.04562v1), 2025-04-06

(83) [Chain of Understanding: Supporting Code Understanding with Large Language Models](http://arxiv.org/abs/2504.04553v1), 2025-04-06

(84) [Advancing Egocentric Video Question Answering with Multimodal Large Language Models](http://arxiv.org/abs/2504.04550v1), 2025-04-06

(85) [The Point, the Vision and the Text: Does Point Cloud Boost Spatial Reasoning of Large Language Models?](http://arxiv.org/abs/2504.04540v1), 2025-04-06

(86) [An Empirical Comparison of Text Summarization: A Multi-Dimensional Evaluation of Large Language Models](http://arxiv.org/abs/2504.04534v1), 2025-04-06

(87) [Trust Region Preference Approximation: A simple and stable reinforcement learning algorithm for LLM reasoning](http://arxiv.org/abs/2504.04524v1), 2025-04-06

(88) [Hessian of Perplexity for Large Language Models by PyTorch autograd (Open Source)](http://arxiv.org/abs/2504.04520v1), 2025-04-06

(89) [Saliency-driven Dynamic Token Pruning for Large Language Models](http://arxiv.org/abs/2504.04514v1), 2025-04-06

(90) [Attributed Synthetic Data Generation for Zero-shot Domain-specific Image Classification](http://arxiv.org/abs/2504.04510v1), 2025-04-06

(91) [Building LLM Agents by Incorporating Insights from Computer Systems](http://arxiv.org/abs/2504.04485v1), 2025-04-06

(92) [VideoAgent2: Enhancing the LLM-Based Agent System for Long-Form Video Understanding by Uncertainty-Aware CoT](http://arxiv.org/abs/2504.04471v1), 2025-04-06

(93) [Domain Generalization for Face Anti-spoofing via Content-aware Composite Prompt Engineering](http://arxiv.org/abs/2504.04470v1), 2025-04-06

(94) [In which fields do ChatGPT 4o scores align better than citations with research quality?](http://arxiv.org/abs/2504.04464v1), 2025-04-06

(95) [An overview of model uncertainty and variability in LLM-based sentiment analysis. Challenges, mitigation strategies and the role of explainability](http://arxiv.org/abs/2504.04462v1), 2025-04-06

(96) [On the Spatial Structure of Mixture-of-Experts in Transformers](http://arxiv.org/abs/2504.04444v1), 2025-04-06

(97) [AGITB: A Signal-Level Benchmark for Evaluating Artificial General Intelligence](http://arxiv.org/abs/2504.04430v1), 2025-04-06

(98) [IntentContinuum: Using LLMs to Support Intent-Based Computing Across the Compute Continuum](http://arxiv.org/abs/2504.04429v1), 2025-04-06

(99) [Driving-RAG: Driving Scenarios Embedding, Search, and RAG Applications](http://arxiv.org/abs/2504.04419v1), 2025-04-06

(100) [Universal Item Tokenization for Transferable Generative Recommendation](http://arxiv.org/abs/2504.04405v1), 2025-04-06
